src/main/java/mods/eln/Eln.java

```diff
@@ -4836,7 +4836,7 @@ private void registerPortableItem(int id) {
 
             BatteryItem desc = new BatteryItem(
                 name,
-                20000, 500, 100,// double energyStorage,double
+                40000, 125, 250,// double energyStorage,double - Haxorian note: doubled storage halved throughput.
```

Comment:
Note to self: modify tooltips. Halving throughput doesn't feel quite right but perhaps I'm biased to the current (heh) configuration of them. I'll playtest soon.

Reply:
Halving the throughput was to offset the halved power draw from power tools, and to prevent the 4battery pack from being too overpowered on E-coal armor. Before, the throughput limits were on the equipment and the battery's throughput was way too high which made the capacitor really trash. Now the capacitor has a home and the battery was made more useful in the place it should have been (storage) and less where it has no right to be (output per second)